regression/cbmc/pragma_cprover1/main.c

```diff
@@ -0,0 +1,16 @@
+int main()
+{
+  int x;
+  int y[1];
+
+  // clang-format off
+  // clang-format wants to put spaces around the hyphens
```

Comment:
This could get tedious... Maybe use a string literal?

Reply:
The syntax now is
```
#pragma CPROVER(push, no_signed_overflow_check)
...
#pragma CPROVER(pop)
```
which a) avoids and clang-format hackery and b) covers the features that @danielsn suggested.